src/main/java/com/startup/models/Teacher.java

```diff
@@ -0,0 +1,91 @@
+package com.startup.models;
+
+public class Teacher {
+
+    private long teacherId;
+    private String firstName;
+    private String lastName;
+    private String gender;
```

Comment:
Should we go on gender with a char type? i.e 'm' for male and 'f' for female?

Reply:
I think we can use in this case ENUM (MALE, FEMALE, NO GENDER etc), what do you think about ?

Reply:
I would love to use enum, so yeah let's do that!